src/readii/negative_controls_refactor/manager.py

```diff
@@ -0,0 +1,121 @@
+from dataclasses import dataclass, field
+from itertools import product
+from typing import Iterator, List, Optional, TypeVar
+
+import numpy as np
+import SimpleITK as sitk
+
+from .abstract_classes import NegativeControlStrategy, RegionStrategy
+from .negative_controls import RandomizedControl, SampledControl, ShuffledControl
+from .regions import FullRegion, NonROIRegion, ROIRegion
+
+T = TypeVar("T", np.ndarray, sitk.Image)
```

Comment:
Can we give this a more descriptive name? Maybe Image_T?

Reply:
yeah we def could, just updated